Task60/src/main/java/driver/Driver.java

```diff
@@ -0,0 +1,26 @@
+package driver;
+
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.chrome.ChromeDriver;
+import java.util.concurrent.TimeUnit;
+
+public class Driver {
```

Comment:
Create singleton for WebDriver

Reply:
getDriver() supposed to be called after open(), otherwise you will have NullPointerException. This is bad code. Singleton will help you to avoid this bottleneck